wikidata/entity.py

```diff
@@ -238,6 +238,13 @@ def attributes(self) -> Mapping[str, object]:
     def load(self) -> None:
         url = './wiki/Special:EntityData/{}.json'.format(self.id)
         result = self.client.request(url)
+        if result is None:
+            self.data = None  # Not loaded yet
+            return
+        elif result == 'Non-Existent':
+            self.data = {}
```

Comment:
I believe there would be a better way to represent non-existent entities than an empty dictionary.  How about `Entity` having an attribute/property like `.state`?  We can define an `enum` type for `Entity.state` like:

```python
class EntityState(enum.Enum):
    not_loaded = 'not_loaded'
    loaded = 'loaded'
    non_existent = 'non_existent'
```